pegasus/sites.v3/code.org/public/districts.haml

```diff
@@ -111,6 +114,20 @@ theme: responsive_full_width
       %a.link-button{href: url_learn_more} Learn more
       %a.link-button.secondary{href: url_enroll_now, target: "_blank", rel: "noopener noreferrer"} Enroll now
 
+%section.bg-neutral-light
+  %h2.centered
+    Meet the Districts in our District Program
+  %p.centered
+    Over 100 school districts have already joined!
+  .logos.flex-container.justify-center.align-items-start.wrap
```

Comment:
You can use the `.logos-wrapper` class here; it should negate the need to add any custom styles, but you will need to set a `max-width` on the images ([see docs](https://docs.google.com/document/d/1wev8I2BQVViNXPuhLEXMpIAYSoAJb0kVoSRKaI3zEAk/edit#heading=h.buqozp3p1hoo))

Reply:
Updated and added screenshots!